tests/lib/rules/no-unpublished-require.js

```diff
@@ -181,7 +182,7 @@ ruleTester.run("no-unpublished-require", rule, {
         {
             filename: fixture("1/test.js"),
             code: "require(`foo${bar}`);",
-            env: { node: true, es6: true },
+            languageOptions: { globals: globals.node },
```

Comment:
**question:** this should not be needed as its in the rule tester defaults (`tests/eslint-rule-tester.js:25`)